script.trakt/resources/lib/deviceAuthDialog.py

```diff
@@ -0,0 +1,59 @@
+import xbmcgui
```

Comment:
`__init__.py` files are not necessary with python3

Reply:
Done